packages/react-core/src/components/Page/Page.tsx

```diff
@@ -78,7 +78,7 @@ export interface PageProps extends React.HTMLProps<HTMLDivElement> {
    * Can add callback to be notified when resize occurs, for example to set the sidebar isNav prop to false for a width < 768px
    * Returns object { mobileView: boolean, windowSize: number }
    */
-  onPageResize?: (object: any) => void;
+  onPageResize?: (object: any) => void | null;
```

Comment:
I *think* this typing would still require a function, but the function could just return `void | null` and that the following type is what you'd really need.
```suggestion
  onPageResize?: ((object: any) => void) | null;
```

Reply:
Oh yeah, you are correct. This just changes the return type